documentation/_Sidebar.asciidoc

```diff
@@ -1,8 +1,26 @@
 == General
 
-== spring
+* link:architecture.asciidoc[Architecture]
+* link:coding-conventions.asciidoc[Coding Conventions]
+* link:guide-structure.asciidoc[Project Structure]
+* link:guide-dependency-injection.asciidoc[Dependency Injection]
+* link:guide-blob-support.asciidoc[BLOB support]
+* link:guide-jpa.asciidoc[JPA]
+* link:guide-jpa-query.asciidoc[Queries]
+* link:guide-repository.asciidoc[Repository]
+* link:guide-dao.asciidoc[DAO]
+* link:guide-jpa-performance.asciidoc[JPA Performance]
+* link:guide-jpa-idref.asciidoc[JPA IdRef]
+* link:guide-transactions.asciidoc[Transaction Handling]
+* link:guide-sql.asciidoc[SQL]
+* link:guide-database-migration.asciidoc[Database Migration]
+* link:guide-logging.asciidoc[Logging]
+* link:guide-monitoring.asciidoc[Monitoring]
+* link:guide-log-monitoring.asciidoc[Log-Monitoring]
+* link:guide-apm.asciidoc[Application Performance Management]
+* link:guide-security.asciidoc[Security]
 
-=== link:architecture.asciidoc[Architecture Overview]
+== spring
```

Comment:
indeed the sidebar is still messed, but also these changes do not completely fix this.
We should address this outside of this PR that is about validation guide...

Reply:
Removed the changes in the sidebar